### PR TITLE
3.4: Rework attribute usage for Smart Proxies (#2067)

### PR DESCRIPTION
**Pull request by maximiliankolb**
Follow-up for #2067 

There were merge conflicts in the following files:

````
	both modified:   guides/common/modules/con_prerequisites-for-configuring-smart-proxy-server-for-load-balancing.adoc
	both modified:   guides/common/modules/proc_configuring-your-deployment-to-run-ansible-roles.adoc
	both modified:   guides/common/modules/proc_disabling-cockpit-on-project.adoc
	both modified:   guides/common/modules/proc_installing-the-load-balancer.adoc
	both modified:   guides/doc-Upgrading_and_Updating/topics/upgrading_overview.adoc
````

I've then checked them out from 3.4 again and rerun all four commands as specified in the original commit message.

### CODE REVIEW

File: guides/doc-Upgrading_and_Updating/topics/upgrading_capsule_server.adoc [hunk 1]
```diff
@@ -33,7 +33,7 @@ If these files have been deleted, they must be restored from a backup in order f
 ====
 endif::[]
 
-.Upgrading {SmartProxyServer}s
+.Upgrading {SmartProxyServers}
```
Comment (adamlazik1) [inline guides/doc-Upgrading_and_Updating/topics/upgrading_capsule_server.adoc:36]: It's probably not stated anywhere, but from what I can tell we also keep unnumbered titles camel-cased?
```suggestion
.Upgrading {SmartProxyServersTitle}
```

Comment (maximiliankolb) [reply guides/doc-Upgrading_and_Updating/topics/upgrading_capsule_server.adoc:36]: Yes, we should do this. I will open a new PR for that against master and then start cherry-picking. This PR is already a cherry-pick from master.

Comment (adamlazik1) [reply guides/doc-Upgrading_and_Updating/topics/upgrading_capsule_server.adoc:36]: Sounds good to me!

**Review by adamlazik1 — APPROVED** [review]

**Comment by maximiliankolb** [comment]
Merged to 3.4 and cherry-picked to 3.3: :heavy_check_mark: 
   ad046ed06..62c47f363  3.3 -> 3.3

**Pull request merged**